exercises/isogram/tests/isogram.rs

```diff
@@ -0,0 +1,62 @@
+extern crate isogram;
+
+use isogram::check;
+
+#[test]
+fn empty_string() {
+    assert_eq!(check(""), true, "An empty string is a isogram.")
+}
+
+#[test]
+fn only_lower_case_characters() {
+    assert_eq!(check("isogram"), true, "\"isogram\" is a isogram.")
+}
+
+#[test]
+fn one_duplicated_character() {
+    assert_eq!(check("eleven"),
+               false,
+               "\"eleven\" has duplicated \'e\' isogram.")
```

Comment:
I don't understand what `duplicated \'e\' isogram.` means. I think it should be `duplicated \'e\' so is not an isogram`, or `duplicated \'e\'`.